examples/using-page-transitions/src/utils/getTransitionStyle.js

```diff
@@ -1,8 +1,6 @@
 const getTransitionStyles = timeout => {
   return {
     entering: {
-      position: `absolute`,
```

Comment:
@stevensurgnier Odd—without these lines, I don't get a fade in.

Reply:
Those css rules were necessary when both elements were on the dom simultaneously. Now one exits (and unmounts) and then the next (mounts) and enters. The trick needed to get the enter animation working is to have different timeouts for enter and exit ([via](https://github.com/fabrictech/gatsby/blob/d1828974e1c79d4282ec8776c2c086655bff4bbb/examples/using-page-transitions/gatsby-browser.js#L51)).